pkg/shp/cmd/buildrun/logs.go

```diff
@@ -73,7 +74,7 @@ func (c *LogsCommand) Run(params *params.Params, ioStreams *genericclioptions.IO
 		return fmt.Errorf("no builder pod found for BuildRun %q", c.name)
 	}
 
-	fmt.Fprintf(ioStreams.Out, "Obtaining logs for BuildRun %q\n\n", c.name)
+	klog.Infof("Obtaining logs for BuildRun %q\n\n", c.name)
```

Comment:
Do we need this printed to stdout? `kubectl logs` doesn't print something like this unless debug/verbose logging is turned on.

Perhaps log at V(5) instead?

Reply:
fixed in next push